szperacz/database.py

```diff
@@ -0,0 +1,20 @@
+
+class Database:
+
+    def __init__(self):
+        self.create()
+        self.read()
+        self.update()
+        self.delete()
+
+    def create(self):
+        create = None
+
+    def read(self):
+        read = None
+
+    def update(self):
+        update = None
+
+    def delete(self):
+        delete = None
```

Comment:
According to task description there should be different body content of the functions:

`Currently the content of the functions can be only appropriate log`

Please fix

Reply:
I don't understand 
"there should be different body content of the functions" - different from each other? 
or all 4 functions should look like this? 

def create(self):
    pass

I understand that in the future these functions are to be connected to the database, but now I do not know completly what should be in the body of the functions